test/sharness/t0088-repo-stat-symlink.sh

```diff
@@ -22,7 +22,8 @@ test_expect_success "'ipfs repo stat' RepoSize is correct with sym link" '
   export IPFS_PATH=".ipfs" &&
   reposize_symlink=$(ipfs repo stat | grep RepoSize | awk '\''{ print $2 }'\'') &&
   echo "reposize_symlink: $reposize_symlink;  reposize_direct: $reposize_direct" &&
-  test $reposize_symlink -ge $reposize_direct
+  reposize_diff="$(( $reposize_symlink - $reposize_direct ))" &&
+  test "${reposize_diff#-}" -lt 2048 # less than 2k diff
 '
```

Comment:
Any particular reason for choosing "2048" as the threshold?

Reply:
1k was too small in my testing.

Reply:
I am thinking the threshold should be larger.  If there was a symbolic link that was not followed then I image either 1) there will be an error or 2) the size will be 0 or something very close to it...

Reply:
Prior to the PR to add support for symbolic links, the RepoSize for a symbolic link .ipfs path was non-zero but small. I don't remember exactly what it was but I think it was <100 bytes.

Reply:
In that case, the correct fix would probably be to just check if the repo-size (as seen via a symlink) is over 1KiB. Sound good?

Reply:
I would test @leerspace assumption first by undue his change and seeing what size is reported.  But otherwise that sounds good.

Reply:
Tested and fixed. I'm now testing to make sure that the repo size is greater than the size of the symlink itself.

Reply:
That seams a little overly specific, but if that the size that was being reported before it LGTM.

Reply:
I could just choose some arbitrary minimum but this is really what we're trying to test.

Reply:
Either way is fine by me.